benchmarks/web-search/client/docker-entrypoint.sh

```diff
@@ -4,7 +4,7 @@
 export IP=$1
 
 #Read local IP
-export HOST_IP=$(echo `ifconfig eth0 2>/dev/null|awk '/inet addr:/ {print $2}'|sed 's/addr://'`) \
+export HOST_IP=$(echo `hostname`) \
```

Comment:
This change was necessary since the ifconfig command does not work in the later versions of Ubuntu and the parsing of the IP address done was not very reliable.